### ISSUE TITLE
Warning: Cannot find Stdlib

### ISSUE THREAD

**Issue by hbr** [ocaml/odoc#825]
When creating my documentation via odoc I usually get the warning that odoc cannot find module `Stdlib`. The reason seams to be that e.g. `String.t` is within the type of a function which is resolved to `Stdlib.String.t` (which is correct), but the documentation for `Stdlib` cannot be referenced.

Is there a way to avoid this warning or link to the documentation of `Stdlib`?

**Comment by Julow** [comment]
The documentation for the stdlib isn't built, Odoc can't reference it. Dune doesn't build the documentation of dependencies and so the documentation is incomplete.

For now, Odig and the new [documentation website](https://v3.ocaml.org/packages) build the doc correctly.

On the other hand, the incomplete documentation is a lot smaller (doesn't contain all the transitive dependencies) and still readable.

**Comment by Julow** [comment]
Agree. It is planned to make these error a lot less verbose. Also, they are not turn into errors with `--warn-error`.
